### DISCUSSION TITLE
Is it possible to synchronise wsEndpoint of already running driver window

### DISCUSSION THREAD

**Discussion by STARKILLER-1** [seleniumbase/SeleniumBase#2738] [Q&A] [answered]
I have like "wsEndpoint":"/devtools/browser/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx"
and sbase is running it on another ws
and i get :
 Message: disconnected: unable to connect to renderer
Stacktrace:
        GetHandleVerifier [0x00007FF6D96E1502+60802]
        (No symbol) [0x00007FF6D965AC02]
        (No symbol) [0x00007FF6D9517CE4]
        (No symbol) [0x00007FF6D9555277]
        (No symbol) [0x00007FF6D954FF39]
        (No symbol) [0x00007FF6D954AA64]
        (No symbol) [0x00007FF6D9596037]
        (No symbol) [0x00007FF6D95956B0]
        (No symbol) [0x00007FF6D958A923]
        (No symbol) [0x00007FF6D9558FEC]
        (No symbol) [0x00007FF6D9559C21]
        GetHandleVerifier [0x00007FF6D99E411D+3217821]
        GetHandleVerifier [0x00007FF6D9A260B7+3488055]
        GetHandleVerifier [0x00007FF6D9A1F03F+3459263]
        GetHandleVerifier [0x00007FF6D979B846+823494]
        (No symbol) [0x00007FF6D9665F9F]
        (No symbol) [0x00007FF6D9660EC4]
        (No symbol) [0x00007FF6D9661052]
        (No symbol) [0x00007FF6D96518A4]
        BaseThreadInitThunk [0x00007FF9D62C4DE0+16]
        RtlUserThreadStart [0x00007FF9D81FEDAB+43]
it sometimes works, sometimes not

**Comment by mdmintz** [comment, marked as answer]
There's a lot of `wsEndpoint` posts on StackOverflow: https://stackoverflow.com/search?q=wsEndpoint
I've never heard of it. Looks like there's a bunch of Puppeteer posts for it.
There's nothing with `wsEndpoint` and `Selenium`: https://stackoverflow.com/search?q=wsEndpoint+Selenium

You didn't show any code, so I have no idea what you did when that error message appeared.
If you're trying to connect to an already-running web browser, see: https://github.com/seleniumbase/SeleniumBase/discussions/2729#discussioncomment-9266099
For debugging, it also helps to know which of the [23 Syntax Formats](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md) you're using.
